docs/bonding-intro.md

```diff
@@ -1,5 +1,4 @@
-What are groups
-===============
+# What are groups #
 
 A Group is an entity that binds multiple sockets and it is required to
```

Comment:
No closing `#` is needed. Just as simple as
`# What are groups`

Reply:
But allowed... :)

Reply:
Indeed. Then put a thousand of them. :)